pilot/pkg/serviceregistry/kube/controller/ambient/authorization.go

```diff
@@ -62,8 +62,20 @@ func (a *index) Policies(requested sets.Set[model.ConfigKey]) []model.WorkloadAu
 	return res
 }
 
+func getOldestPeerAuthn(policies []*securityclient.PeerAuthentication) *securityclient.PeerAuthentication {
+	var oldest *securityclient.PeerAuthentication
+	for _, pol := range policies {
+		if oldest == nil || pol.CreationTimestamp.Before(&oldest.CreationTimestamp) {
+			oldest = pol
+		}
+	}
+	return oldest
+}
+
 // convertedSelectorPeerAuthentications returns a list of keys corresponding to one or both of:
 // [static STRICT policy, port-level STRICT policy] based on the effective PeerAuthentication policy
+// TODO: If the workload peerauthn has a mode UNSET and a parent-level strict policy, merge and don't attach
```

Comment:
Why? it seems right as it is, sending the strict policy?

Reply:
Yeah this is incorrectly phrased, but also implemented (correctly). What I mean to say was "if the workload peerauthn has a mode UNSET with a port-level PERMISSIVE and a parent-level strict policy, merge and don't send the static strict pol". I just forgot to come back and clean this up